JASP-Desktop/analysis/analysis.cpp

```diff
@@ -856,6 +856,7 @@ void Analysis::fitOldUserDataEtc()
 		if(orphans.size() > 0)
 			_userData["orphanedNotes"] = orphans;
 
+		_userData["fitOldUserDataEtc"] = true;
```

Comment:
And please add comments when you do non-obvious things like this. If we look at this code in a year, would you expect to know what is going on here?

Reply:
If you want to keep it likes this (as opposed to my suggestion) then maybe rename this property to "processThisUserdata" or something. Because "fitOldUserDataEtc" is not very descriptive.

Reply:
And then we could reuse it later as well in other places.